ophyd/status.py

```diff
@@ -26,6 +25,14 @@ def f(self, *args, **kwargs):
     return f
 
 
+class StatusBaseLoggerAdapter(LoggerAdapter):
+    """
+    A LoggerAdapter for use by StatusBase and derived classes.
+    """
+    def process(self, msg, kwargs):
+        return f"[{str(self.extra['status'])}] {msg}", kwargs
```

Comment:
Same comments as above regarding using `Formatter` instead of a custom subclass.